rc/pdfpcrc

```diff
@@ -1,129 +1,123 @@
+bind space              next
+bind Down               next
+
+# Standard backward movement
+bind Prior              prev
+bind Left               prev
+bind Up                 prev
+bind BackSpace          prev
+
+# Overlay paging
+bind S+Next             nextOverlay
+bind S+Prior            prevOverlay
+
+# Intra-slide navigation
+bind S+Up               lastOverlay
+bind S+Down             firstOverlay
+
+# Fast forward/backward (10 pages)
+bind S+Right            next10
+bind S+Left             prev10
 
 # "Jumps"
-bind Home      gotoFirst
-bind End       gotoLast
-bind g         goto
-bind Tab       overview
-bind BackSpace histBack
+bind Home               gotoFirst
+bind End                gotoLast
+bind g                  goto
+
+# Bookmarking [the current action names are misleading]
+bind m                  lastSlide
+bind S+m                jumpLastSlide
+
+# History? [these 3 are seriously broken]
+bind S+BackSpace        histBack
+bind S+End              nextUnseen
+bind S+Home             prevSeen
+
+# The overview mode
+bind Tab                overview
+
+# Toolbox
+bind t                  toggleToolbox
 
 # Exit
-bind Escape    quit
-bind q         quit
+bind C+q                quit
+
+# Presentation state
+bind b                  blank
+bind f                  freeze
+bind h                  hide
+bind Escape             exitState
 
 # Timer related
-bind s         start
-bind p         pause
-bind r         resetTimer
-bind R         reset
-
-# Change state
-bind b         blank
-bind period    blank
-bind f         freeze
-bind h         hide
-
-# Presentation properties
-bind n         note
-bind o         overlay
-bind e         endSlide
-bind S+s       lastSlide
-bind S+r       jumpLastSlide
+bind s                  start
+bind p                  pause
+bind C+t                resetTimer
 
-# Annotation modes
-bind 0         switchMode normal
-bind KP_0      switchMode normal
-bind 2         switchMode eraser
-bind KP_2      switchMode eraser
-bind 5         switchMode pen
-bind KP_5      switchMode pen
-bind 8         switchMode pointer
-bind KP_8      switchMode pointer
+# Presentation editing
+bind C+n                note
+bind C+o                overlay
+bind C+e                endSlide
 
-# Toolbox
-bind t toggleToolbox
-
-# Font scaling
-bind plus      increaseFontSize
-bind equal     increaseFontSize
-bind KP_Add    increaseFontSize
-bind bracketleft increaseFontSize
-bind minus     decreaseFontSize
-bind KP_Subtract decreaseFontSize
-bind bracketright decreaseFontSize
-
-# Pointer
-bind 7         decreasePointer
-bind KP_7      decreasePointer
-bind 9         increasePointer
-bind KP_9      increasePointer
-
-# Drawing
-bind 6         clearDrawing
-bind KP_6      clearDrawing
-bind 4         toggleDrawings
-bind KP_4      toggleDrawings
-
-bind 1         decreasePen
-bind KP_1      decreasePen
-bind 3         increasePen
-bind KP_3      increasePen
+# Reset the presentation
+bind C+z                reset
```

Comment:
This seems artificial but I have no better suggestion. Does it actually do what is proposes? Does it reset all states (pen, eraser, colors of them, font size changes)?

Reply:
I agree it's artificial. It was late night and I couldn't find anything better :). Please suggest something more reasonable, really.
I think it only resets timer and moves to the first page. We can add exiting, if needed, freeze/hidden/etc as well. Yes, probably resetting sizes/colors makes sense as well.

Reply:
I was talking about deleting it completely. Since we are not resetting the state completely, this function is "buggy" and I also don't find it useful. You can achieve the same effect with Quit-and-restart.